centos_install_requirements.sh

```diff
@@ -77,6 +77,19 @@ sudo yum -y install \
   unzip \
   genisoimage
 
+if [ "${CONTAINER_RUNTIME}" == "podman" ]; then
```

Comment:
it should  be either 
[[ "${CONTAINER_RUNTIME}" == "podman" ]]
or
[ "${CONTAINER_RUNTIME}" = "podman" ]

Reply:
Fixed in three files